### PR TITLE
Fix crash in support design library(v.23.2.+)

### PR DESCRIPTION
**Pull request by h3r3x3**
Fix "View can not be anchored to the the parent CoordinatorLayout" error in new version support design library(v.23.2.+)

### CODE REVIEW

File: todoapp/build.gradle [hunk 1]
```diff
@@ -23,13 +23,13 @@ task clean(type: Delete) {
 // Define versions in a single place
 ext {
     // Sdk and tools
-    minSdkVersion = 10
+    minSdkVersion = 14
```
Comment (malmstein) [inline todoapp/build.gradle]: @JoseAlcerreca do we care about this?

Comment (h3r3x3) [inline todoapp/build.gradle]: Wow, it's my mistake (×_×)

Comment (JoseAlcerreca) [inline todoapp/build.gradle]: Yes we do.

**Comment by JoseAlcerreca** [comment]
Sorry I didn't approve this PR earlier. There's a problem with the FAB not going back down after a Snackbar is shown, but it turns out it's a bug in the support lib. This happens only when animations are disabled, so this PR LGTM.

**Pull request merged**